### ISSUE TITLE
Add initial maverick configuration ui 

### ISSUE THREAD

**Issue by SamuelDudley** [goodrobots/maverick-web#142]
- [ ] interface for maverick commands, start, stop, etc...
- [ ] interface to view service status
- [ ] interface to view and edit config files

**Comment by SamuelDudley** [comment]
How vscode handles configuration from a form perspective (which in the background is a json file)
![image](https://user-images.githubusercontent.com/8290680/73407125-40aee280-434c-11ea-93e0-456d6d1b0f52.png)

**Comment by SamuelDudley** [comment]
Basic interface to view and edit configuration files is complete. Working on a UI for commands and system processes now.

**Comment by SamuelDudley** [comment]
https://github.com/xtermjs/xterm.js
Use terminal for maverick command feedback

**Comment by fnoop** [comment]
ooh xterm.js is a nice idea!

**Comment by SamuelDudley** [comment]
added the -web component of xterm.js https://github.com/goodrobots/maverick-web/commit/dc27bb11fe104abbde747e6f53bb9de513c84d5c
just hacked it in, but we can link this to -api for two outcomes:

1. Show realtime terminal output from the process runner (e.g. `maverick configure`)
https://github.com/goodrobots/maverick-api/blob/7ef29f4283e618320710c825c3b21513ab62bb96/maverick_api/modules/base/util/process_runner.py

2. Allow the user to interact with the terminal from the webui using the process runner

**Comment by SamuelDudley** [comment]
https://github.com/wavesoft/local-echo/blob/master/lib/LocalEchoController.js

**Comment by fnoop** [comment]
awesome!  would be great to have a mav terminal built in

**Comment by SamuelDudley** [comment]
The terminal support was working (ish) prior to the API refactor in -web. It would be nice to get that back as it saves a user from needing to SSH in for basic stuff.

**Comment by fnoop** [comment]
'Commands' re-instated.  If it's going to be a full terminal (which is great), should probably be renamed to Console/Terminal.
Console works-ish, restricted to a particular directory and the output is a bit hoky.  If we can get it working better will be fantastic to have.

**Comment by SamuelDudley** [comment]
Yeah, the plan is to have the full terminal working. -api is okay at this stage, the remaining work is in the -web component.
